BrainPortal/app/views/custom_filters/_new.html.erb

```diff
@@ -25,23 +25,26 @@
 <% custom_filter ||= @custom_filter %>
 <% filter_type = custom_filter.class.name.underscore %>
 
-<h2>New Custom Filter</h2>
+<h2> <%= filter_type.capitalize.gsub("_"," ")%> </h2>
 
-<div class="generalbox">
+<%= form_for @custom_filter, :as => :site, :url => { :action => "create" }, :datatype => "script" do |f| -%>
 
-  <%= ajax_form_for custom_filter, :as => :custom_filter, :url => {:controller  => "custom_filters", :action => "create" }, :datatype  => "script" do |f| %>
-    <%= hidden_field_tag :filter_class, filter_type %>
-    <%= error_messages_for(custom_filter, :object_name => "custom filter") %>
+  <div class="generalbox">
 
-    <p>
-    <%= f.label :name, "Filter name" %><br>
-    <%= f.text_field :name %>
+    <%= ajax_form_for custom_filter, :as => :custom_filter, :url => {:controller  => "custom_filters", :action => "create" }, :datatype  => "script" do |f| %>
```

Comment:
And here you have another form embedded the first form? That's illegal

Reply:
Please double-check the structure of the document with View Source because I found many errors...